launcher/Application.cpp

```diff
@@ -1547,7 +1547,10 @@ QString Application::getJarPath(QString jarFile)
         FS::PathCombine(m_rootPath, "share/" + BuildConfig.LAUNCHER_APP_BINARY_NAME),
 #endif
         FS::PathCombine(m_rootPath, "jars"),
-        FS::PathCombine(applicationDirPath(), "jars")
+        FS::PathCombine(applicationDirPath(), "jars"),
+#if !defined(NDEBUG)
```

Comment:
I think we can keep this in in general. I don't really like how `NDEBUG` works, so let's just keep this in all builds

Reply:
:+1: